packages/spark-core/objects/_stack.scss

```diff
@@ -141,6 +192,46 @@
       margin-right: $stack-spacing-huge;
     }
 
+    > .sprk-o-Stack__item--sixth\@s {
+      flex: 0 0 16.666666667%;
```

Comment:
Not suggesting we change anything, but have we thought about using calc for stuff like this?

Reply:
@yodasw16, interesting idea! No, I didn't think about using it. I strayed away from the abstraction because I thought seeing the output of --sixth vs. seeing `calc(...)` in the dev tools was more useful when putting things together.

Reply:
I'm not sure it even matter since the browser just rounds things how it wants to anyway.